libandroid-navigation/build.gradle

```diff
@@ -57,6 +57,7 @@ android {
 
 dependencies {
   api dependenciesList.mapboxSdkServices
+  api dependenciesList.mapboxSdkDirectionsModels
```

Comment:
Do we need this dependency? It should come together with `mapboxSdkServices`.

Reply:
Yeah, agreed. Did you add this for testing purposes (explicitly bring the `SNAPSHOT` version in `mapboxSdkDirectionsModels`) @korshaknn? If so, we should remove before merging the PR 👍

Reply:
fixed